### ISSUE TITLE
Media library not refreshing after uploading via WP media modal

### ISSUE THREAD

**Issue by csossi** [GoogleForCreators/web-stories-wp#4109]
## Bug Description

A newly uploaded image is appearing immediately in "Images" media category, but not in "All" category

## Expected Behaviour

Expected newly uploaded image to appear under both "Images" and "All"

## Steps to Reproduce

- create new story
- select "Upload" from media panel
- browse for and upload an image
- insert this new image into page

- note that image isn't appearing under "All" category (even though it IS appearing in "Images" category

## Screenshots

Newly uploaded image is on the canvas here, but isn't in media "all" section:
![image](https://user-images.githubusercontent.com/16945664/90705690-8936da80-e261-11ea-9472-0a412049589d.png)

However, you can see it here when viewing just "Images":
![image](https://user-images.githubusercontent.com/16945664/90705757-abc8f380-e261-11ea-8bd1-666302ddefeb.png)

(Note: If I navigate away from editor and return, then the image WILL appear under "All")

## Additional Context

- Plugin Version: Testing in PR#4012
- Operating System: Win10
- Browser: Chrome

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
Seems like a duplicate of #1201 — either one should be closed. Need to check with media pod though.

**Comment by swissspidy** [comment]
Oh, and not a stable blocker IMO

**Comment by bmattb** [comment]
@o-fernandez  - can you confirm that this is not a blocker to Beta 2 release today

**Comment by o-fernandez** [comment]
Agree, not a blocker for Beta. But let's take a look at it for Stable. @diegovar would it be possible to TAL at this?

**Comment by swissspidy** [comment]
Estimate seems a bit high. Perhaps @spacedmonkey could be of assistance here regarding the media modal?

**Comment by ezxyz** [comment]
See: https://xwp.slack.com/archives/CCCAUAH6F/p1598962509042200 for discussion.

TLDR: This is possibly due to a PHP issue, but would like someone who knows PHP to confirm.

**Comment by ezxyz** [comment]
So the conclusion from the discussion is that it is because the initial request is being cached so we just need to add a parameter like `cache_bust=true` to the api request for 'All" media.
